README.md

```diff
@@ -110,6 +110,32 @@ Please check `bump deploy --help` for more usage details
 
 _If you want to receive automatic `bump diff` results on your Github Pull Requests you might be interested by [our Github Action](https://github.com/marketplace/actions/api-documentation-on-bump#api-diff-on-pull-requests) diff command._
 
+#### Public API diffs
+
+From any two API definition files or URLs you can retrieve a comprehensive changelog of what has changed between the two given definitions.
```

Comment:
What about:
`From any two definition files or URLs, you can retrieve a comprehensive changelog of what has changed between them.` ?

Reply:
```suggestion
From any two definition files or URLs, you can retrieve a comprehensive changelog of what has changed between them.
```
Indeed, thanks!